src/index.js

```diff
@@ -376,7 +374,7 @@ const initialize = async () => {
       multisigContractStatus.innerHTML = 'Deployed';
       sendMultisigButton.disabled = false;
       // ERC721 Token - Collectibles contract
```

Comment:
Nit: One leftover reference to "collectibles" here
```suggestion
      // ERC721 Token - NFT contract
```